test/error.ts

```diff
@@ -193,6 +193,12 @@ test('errors are thrown directly when options.stream is true', t => {
 	});
 });
 
+test('normalization errors using convenience methods', async t => {
+	const error = await t.throwsAsync(got('undefined/https://example.com'));
```

Comment:
```suggestion
	await t.throwsAsync(got('undefined/https://example.com').json().text().buffer(), 'Invalid URL');
```

Reply:
Fixed!

Reply:
You forgot to test `.text()` and `.buffer()`. Make sure that the error message is `Invalid URL`. As in the suggestion.

Reply:
Ah, I must have missed the `Invalid URL` part. Do you want a separate assertion for each method?

Reply:
No need to